### PR TITLE
Update changelog

### PR DESCRIPTION
**Pull request by ratik21**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,9 +1,23 @@
 # CHANGELOG
 
-## unreleased
-
-+ Updating the default token and endpoint port. For compatibility with Sandbox we use the sandbox token and port (4001) in all examples and sample project. If you have your run an algorand node using our private node setup then either recreate the network (stop, remove node_data and create it again), or update the `node_data/PrimaryNode/config.json` and set: `"EndpointAddress": "127.0.0.1:4001"`
-
+## v0.5.0
+
+* Full support for asset related transaction (create, opt-in, transfer, modify, freeze, revoke)
```
Comment (amityadav0) [inline CHANGELOG.md:5]:
```suggestion
* Full support for asset related transaction (create, opt-in, transfer, modify, freeze, revoke, destroy)
```

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,9 +1,23 @@
...
+* Support for group transactions
+* Support Sandbox in `/infrastructure` to quickly set up the private net
+* Add documentation (in `/docs/testing-teal.md`) to test TEAL using `@algorand-builder/runtime`
+* Updating the default token and endpoint port. For compatibility with Sandbox we use the sandbox token and port (4001) in all examples and sample project. If you have your run an algorand node using our private node setup then either recreate the network (stop, remove node_data and create it again), or update the `node_data/PrimaryNode/config.json` and set: `"EndpointAddress": "127.0.0.1:4001"`
```
Comment (amityadav0) [inline CHANGELOG.md:9]:
```suggestion
* Updating the default token and endpoint port. For compatibility with Sandbox we use the sandbox token and port (4001) in all examples and sample project. If you run an algorand node using our private node setup then either recreate the network (stop, remove node_data and create it again), or update the `node_data/PrimaryNode/config.json` and set: `"EndpointAddress": "127.0.0.1:4001"`
```

File: CHANGELOG.md [hunk 3]
```diff
@@ -1,9 +1,23 @@
...
+
+## v0.4.0 2021-02-03
+
+* Renamed `@algorand-builder/algorand-js` to `@algorand-builder/runtime`
+* Added new example project - Crowdfunding Application
+* Added transaction in `@algorand-builder/runtime` for app creation, opt-in
+* Added support for arguments in stateful smart contracts similar to goal (eg. `str:abc`)
```
Comment (amityadav0) [inline CHANGELOG.md:16]:
```suggestion
* Added support for arguments in stateful smart contracts similar to goal (eg. `str:abc`, 'int:12')
```

File: CHANGELOG.md [hunk 4]
```diff
@@ -1,9 +1,23 @@
 # CHANGELOG
 
-## unreleased
-
-+ Updating the default token and endpoint port. For compatibility with Sandbox we use the sandbox token and port (4001) in all examples and sample project. If you have your run an algorand node using our private node setup then either recreate the network (stop, remove node_data and create it again), or update the `node_data/PrimaryNode/config.json` and set: `"EndpointAddress": "127.0.0.1:4001"`
-
+## v0.5.0
```
Comment (robert-zaremba) [inline CHANGELOG.md:3]:
```suggestion
## unreleased
```

**Review by robert-zaremba — APPROVED** [review]
let's use unreleased for not released changelog entries

**Pull request merged**